### ISSUE TITLE
[QUESTION] Is it normal for it to be slow?

### ISSUE THREAD

**Issue by masterofobzene** [Somnifer2000/Imagefap-Gallery-Downloader#3]
Sorry but can't ask this to anyone else. Is it normal for the script to be really slow?
The script works as expected but 2x or 3x times slower than the gif in the readme.

**Comment by Somnifer2000** [comment]
Yeah, pretty much. The actual speed depends mainly on 2 factors: the size of the gallery (especially the number of pages), and the size of the images themselves. Both can slow down the script in two separate stages.
Basically it will take longer to download a 10-page gallery with extra-high quality images than a 1-page gallery with a only few 500 kB images... 
Hope this answers your question :-)

**Comment by masterofobzene** [comment]
OK! thanks. 
I have been downloading 2 or maybe 3 pages galleries. In the "large" and "huge" realm.
Each photo on discovery/download takes like 3 seconds or so.

**Comment by Somnifer2000** [comment]
Wow, that is frustratingly slow. I'll have a second look at the code, and see what I can come up with. 
Question: is the "Downloading HI-RES IMAGES............"- fase just as slow?

**Comment by masterofobzene** [comment]
YEP... all the process look like that gif. Thanks!
